hack/update-guard-service.sh

```diff
@@ -0,0 +1,2 @@
+export KO_DOCKER_REPO=us.icr.io/knat
+ko apply -Rf ./config/deploy-guard-service.yaml
```

Comment:
If it's just a single file, we can drop the recursive flag. Also might be worth making a separate directory for this file (maybe `config/deploy`) and then can just run `ko apply -f config/deploy`).